src/SortingAlgorithms.jl

```diff
@@ -53,8 +53,8 @@ end
 uint_mapping(::ForwardOrdering, x::Float32)  = (y = reinterpret(Int32, x); reinterpret(UInt32, ifelse(y < 0, ~y, xor(y, typemin(Int32)))))
 uint_mapping(::ForwardOrdering, x::Float64)  = (y = reinterpret(Int64, x); reinterpret(UInt64, ifelse(y < 0, ~y, xor(y, typemin(Int64)))))
 
-uint_mapping{Fwd}(rev::ReverseOrdering{Fwd}, x) = ~uint_mapping(rev.fwd, x)
```

Comment:
`Fwd` isn't used at all. Below, `x::Real` would be enough.

Reply:
would it be
`uint_mapping(rev::ReverseOrdering{Fwd}, x::Real) = ~uint_mapping(rev.fwd, x)`?

Reply:
That would give:
```julia
uint_mapping(rev::ReverseOrdering, x) = ...
uint_mapping(::ReverseOrdering{ForwardOrdering}, x::Real) = ...
```

Reply:
Thanks!

Reply:
But your new commit doesn't implement that...

Reply:
You are correct. Fixing it now. Should the first line specify `x::Real` as well?

Reply:
I have no idea, but I don't see any reason to change that.

Reply:
It should not.  Although uncommon, the ordering object wrapped in the `ReverseOrdering` could potentially change the type of the elements being compared.